MANIFEST.in

```diff
@@ -1,9 +1,4 @@
-include MANIFEST.in
-include LICENSE
 include RELEASE.md
-include README.md
-include setup.py
-include pyproject.toml
```

Comment:
According to https://packaging.python.org/guides/using-manifest-in/, those are included by default.